### ISSUE TITLE
About SimpleUncertainL1Loss

### ISSUE THREAD

**Issue by DaDaDaBiuBiu** [jiayisong/DA3D#3]
Thanks to the author for sharing such an amazing work.
I really want to know why the **SimpleUncertainL1Loss** in the article is 

![image](https://github.com/user-attachments/assets/cc195e4f-ceb1-4bcf-83f9-fd990e46db36)  ,
but in the code it's 
![image](https://github.com/user-attachments/assets/5e03092b-655d-4e02-8a8c-95f7e7e46898)

**Comment by jiayisong** [comment]
Because the derivative of the ln(x) function is 1/x, you can deduce the backpropagation of ln(x) to understand.

**Comment by jiayisong** [comment]
This approach avoids the problem of gradient vanishing when the loss is less than 0.01 in loss=torch.log(loss.clamp_min(0.01)).
